src/controllers/subscriptiontopic_controller.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+const { StatusCodes } = require('http-status-codes');
+const { loggers } = require('@asymmetrik/node-fhir-server-core');
+const { v4: uuidv4 } = require('uuid');
+const db = require('../storage/DataAccess');
+
+const logger = loggers.get('default');
+const SUBSCRIPTION_TOPIC = 'subscriptiontopics';
+
+let build = function (server) {
+  let router = new express.Router();
+  const default_cors_options = Object.assign({}, server.config.server.corsOptions);
+  router.use(bodyParser.json({ type: ['application/json', 'application/fhir+json'] }));
+  router.post('/', cors(default_cors_options), function (req, res) {
```

Comment:
It's a 500 error. Not a huge deal since it works when the user does the right thing, but kind of confusing and unhelpful. We can worry about that later